statsmodels/tsa/statespace/tests/test_mlemodel.py

```diff
@@ -597,6 +597,23 @@ def test_summary():
         res.summary()
 
 
+def test_summary_rsquared():
+    from statsmodels.regression.linear_model import OLS
+    endog = np.array([1, 2, 4, 8, 16])
+    exog = np.array([1, 2, 4, 8, 16])
+
+    mod = sarimax.SARIMAX(endog, exog, order=(0, 0, 0), trend='c')
```

Comment:
Add `concentrate_scale=True` so you don't have to estimate the variance parameter.